system/micd.py

```diff
@@ -16,7 +16,7 @@
 def calculate_spl(measurements):
   # https://www.engineeringtoolbox.com/sound-pressure-d_711.html
   sound_pressure = np.sqrt(np.mean(measurements ** 2))  # RMS of amplitudes
-  if sound_pressure > 0:
+  if sound_pressure >= REFERENCE_SPL:
```

Comment:
```suggestion
  if sound_pressure > REFERENCE_SPL:
```

if equal, it will be 0 anyway

Reply:
True, we could also try 
```suggestion
  sound_pressure_level = 20 * np.log10(max(1, sound_pressure / REFERENCE_SPL))
```
to keep it short